README.md

```diff
@@ -1,5 +1,18 @@
 ### Arrays
 
+#### First
+
+* Underscore
+  ```javascript
+  _.first([5, 4, 3, 2, 1]);
+  ```
+
+* ES2015
+
+  ```javascript
+  const [n] = [5, 4, 3, 2, 1];
+  ```
+
```

Comment:
I would use just `array[0]` as the "vanilla" example, but you probably wouldn't use `_.first` in a simple case like that anyway. (It's more useful for function composition.)

These are also not equivalent, the second one assigns it to a constant, while the first one is just the expression.